pkg/prove/proof_test.go

```diff
@@ -172,14 +173,13 @@ func TestTxShareIndex(t *testing.T) {
 // stripCompactShares strips the universal prefix (namespace, info byte, sequence length) and
 // reserved bytes from a list of compact shares and joins them into a single byte
 // slice.
-func stripCompactShares(compactShares []shares.Share, start uint64, end uint64) (result []byte) {
-	for i := start; i <= end; i++ {
-		if i == 0 {
-			// the first compact share includes the sequence length
-			result = append(result, compactShares[i][appconsts.NamespaceSize+appconsts.ShareInfoBytes+appconsts.SequenceLenBytes+appconsts.CompactShareReservedBytes:]...)
-		} else {
-			result = append(result, compactShares[i][appconsts.NamespaceSize+appconsts.ShareInfoBytes+appconsts.CompactShareReservedBytes:]...)
+func stripCompactShares(compactShares []shares.Share, start uint64, end uint64) (result []byte, err error) {
```

Comment:
looks like start and end are no longer used. Can we remove them?

Reply:
second this, otherwise LGTM and is ready to ship